client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -152,32 +152,35 @@ export class SeoForm extends React.Component {
 		}
 
 		// Don't update state for fields the user has edited
-		nextState = omit( nextState, dirtyFields.toArray() );
+		nextState = omit( nextState, [ ...dirtyFields ] );
 
 		this.setState( {
```

Comment:
this is pre-existing, but `this.setState( nextState )` should be good enough

Reply:
simplified in 58553611a4